### PR TITLE
initial documentation updates

### PR DESCRIPTION
**Pull request by madduran**
The following tools are updated: 
ViewSam
SetNmAndUqTags
SetNmMdAndUqTags
SplitSamByLibrary
VcfToIntervalList

### CODE REVIEW

File: src/main/java/picard/sam/SetNmMdAndUqTags.java [hunk 1]
```diff
@@ -40,27 +40,61 @@
 import java.util.stream.StreamSupport;
 
 /**
+ * Fixes the NM, MD, and UQ tags in a SAM or BAM file.
+ *
+ * <p>This tool takes in a SAM or BAM file (sorted by coordinate) and calculates the NM, MD, and UQ
```
Comment (ktibbett) [inline src/main/java/picard/sam/SetNmMdAndUqTags.java]: consider "coordinate0sorted SAM or BAM file" instead...

File: src/main/java/picard/sam/SetNmMdAndUqTags.java [hunk 2]
```diff
@@ -40,27 +40,61 @@
...
+ * tags by comparing with the reference. </p>
+ *
+ * <p>This may be needed when MergeBamAlignment was run with SORT_ORDER different from 'coordinate'
+ * and thus could not fix these tags then. The input must be coordinate sorted in order to run.
+ * If specified, the MD and NM tags can be ignored and only the UQ tag be set.</p>
+ *
+ * <h3>Input</h3>
+ * <p>
+ * The BAM or SAM file to fix
+ * </p>
```
Comment (ktibbett) [inline src/main/java/picard/sam/SetNmMdAndUqTags.java]: Shouldn't the reference be noted as a required input?

File: src/main/java/picard/sam/SetNmMdAndUqTags.java [hunk 3]
```diff
@@ -40,27 +40,61 @@
 import java.util.stream.StreamSupport;
 
 /**
+ * Fixes the NM, MD, and UQ tags in a SAM or BAM file.
+ *
+ * <p>This tool takes in a SAM or BAM file (sorted by coordinate) and calculates the NM, MD, and UQ
+ * tags by comparing with the reference. </p>
+ *
+ * <p>This may be needed when MergeBamAlignment was run with SORT_ORDER different from 'coordinate'
```
Comment (ktibbett) [inline src/main/java/picard/sam/SetNmMdAndUqTags.java]: consider "other than" rather than "different from"

**Comment by ktibbett** [comment]
Some minor wordsmithing comments and one required omitted required parameter.

👍

**Comment by madduran** [comment]
@ktibbett thanks for the review, I responded to your comments. I also realized that I forgot to add my changes to ViewSam.

**Comment by ktibbett** [comment]
@madduran , back to you for one last round, and then I'll merge.

**Comment by madduran** [comment]
@ktibbett thanks again, comments should all be addressed

**Pull request merged**